tools/bootstrap_upip.sh

```diff
@@ -0,0 +1,13 @@
+# This script performs bootstrap installation of upip package manager from PyPI
+# All the other packages can be installed using it.
+
+# Remove any stale old version
+rm -rf micropython-upip-*
```

Comment:
I think that would be better than doing it in the current directory.  Or force the user to provide a path (eg . or /tmp) to extract the tgz.